README.md

```diff
@@ -20,7 +20,7 @@ It shows that throughput of 10'000 transactions per second is reached
 even in a fully decentralized network with thousands of nodes.
 
 An easy-to-read blog post introduction with videos is written
```

Comment:
Hello,
Thanks for your contribution ❤️ 
Can you please change this line also to match the fact that you know redirect to a video instead of a blog post ?
Thanks you !

Reply:
sure :)